README.md

```diff
@@ -1,83 +1,36 @@
-# Ruby on Rails Course
+# Blograils
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+> This is a blog app created in Microverse Ruby on Rails module.
```

Comment:
Please add a description here explaining what this project all about and what are the functionalities of the App. 👍🏻

Reply:
Hello @zainsadaqat  thank you for the good code review.